### DISCUSSION TITLE
This library is super awesome!

### DISCUSSION THREAD

**Discussion by dortzur** [juliangarnier/anime#977] [General]
Thank you creating it.

**Comment by glomotion** [comment]
+1

**Comment by tinchox5** [comment]
+1

**Comment by bonatoc** [comment]
Ah! You beat me to it.
Also chiming in, and the new docs are awesome.

**Comment by luglio7** [comment]
+1

**Comment by pakx** [comment]
Indeed! Many thanx, Julian! 👍

**Comment by whyboris** [comment]
🎉  2018 and it's still awesome 😉

**Comment by Kihoru** [comment]
+1 with love

**Comment by kellymears** [comment]
+1

**Comment by maisonm** [comment]
+1

**Comment by zzseba78** [comment]
+1
Amazing job, thank you Julian!

**Comment by whyboris** [comment]
🎉 2019 and it's still awesome 😉

**Comment by VulumeCode** [comment]
And the docs are beautiful!

**Comment by sadeghbarati** [comment]
lightweight and free thanks for it 😍

**Comment by juliangarnier** [comment]
Thanks everyone, positive comments like these really mean a lot!
I'm cleaning up the issues now that v4 is out, but I feel bad closing this one, so I'm going to convert it into a discussion instead.❤️

**Comment by empty916** [comment]
Sorry, GSAP. I've fallen for Anime.js.🤣
